docs/CommonQueryPatterns.md

```diff
@@ -58,4 +58,13 @@ otherwise, an empty peer container is created
   - otherwise, global namespaces will be used if existed else cluster has empty namespaces container
 - If any specific key is specified it will override the relevant contents in resourceList
 
+##### Simulating live cluster missing resources:
```

Comment:
Is this the right place to specify this ?
I think maybe we should create another md file for this, and point to it from the main documentation file?

Reply:
done